target_chains/solana/pyth_solana_receiver_state/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+use anchor_lang::declare_id;
+
+pub mod config;
+pub mod price_update;
+
+declare_id!("rec5EKMGg6MxZYaMdyBfgwp4d5rB9T1VQH5pJv5LtFJ");
```

Comment:
I forget how this declare_id thing works but I presume this definition is required here as well as in target_chains/solana/programs/pyth-solana-receiver/src/lib.rs ?

I note that the same string is declared in both places, maybe worth extracting it to a constant?

Reply:
Done